docs/references/api-reference/C_API.md

```diff
@@ -21,7 +21,7 @@
 
 ## Enums
 
-```{eval-rst}
```

Comment:
This change breaks the C API docs page. It needs to be `{eval-rst}` to interpret the block using the rst parser.

Reply:
is this the only one of these?  I will change the generation code so it won't happen again.

Reply:
I updated it

Reply:
There are several other instances throughout this file that also need to get changed back to `{eval-rst}`.

Reply:
There are several other instances throughout this file that also need to get changed back to `{eval-rst}`.

Reply:
ok, think I got it now